features/storage/kvstore/conf/global/mbed_lib.json

```diff
@@ -20,6 +20,9 @@
         "NUCLEO_F429ZI": {
             "storage_type": "TDB_INTERNAL"
         },
+        "DISCO_H747I": {
+            "storage_type": "TDB_INTERNAL"
```

Comment:
Maybe it will be easier to set the default value to TDB_INTERNAL ?
instead of overriding each target one by one ?

Reply:
@SeppoTakalo  - we should consider this?

Reply:
Yes, apparently many modules are assuming that KVStore "just works", so we need to probably default to TDB_INTERNAL and silently steal some flash away from the app.

But of course it needs to be configurable, so that location can be overwritten.